guides/common/modules/con_facts-and-nic-filtering.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: CONCEPT
+
+[id="con_facts-and-nic-filtering_{context}"]
+= Facts and NIC filtering
+
+Facts are host parameters such as total memory, operating system version, or architecture.
+You can find Facts in *Monitor* > *Facts* and search hosts via facts or use facts in templates.
+
+In {Project}, collect facts from following various sources:
+
+* `subscription manager`
+* `ansible`
+* `puppet`
+
+{Project} is an inventory system for hosts and network interfaces.
+For hypervisors or container hosts, it does not make any value to add thousands of interfaces per host into the inventory and update it every few minutes.
+For each individual NIC reported, Foreman creates a NIC entry and those entries are never removed from the database.
+Parsing all the facts and comparing all records in the database makes {Project} extremely slow and unusable.
+To optimize the performance of various actions, most importantly fact import, use options available in *Administer* > *Settings* > *Facts*.
```

Comment:
This would be the `*Facts* tab`. We generally say "In the {ProjectWebUI}, navigate to *X > Y*. On the *Z* tab, selct ...". Maybe it's a good idea to just say that you have change options and then link the appropriate procedure if we have one?